MANIFEST.in

```diff
@@ -1,10 +1,92 @@
-prune .github
-prune requirements
+# A generic template for "MANIFEST.in" files in Scitools repos.
+# NOTE: most of this can't be duplicated as-is into any given repo, but suggests
+#     guidelines + structure.
+# For the syntax of "MANIFEST.in" files, see:
+#     https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#using-manifest-in
+
+# General principles
+#     - enable user to build locally, as well as in CI
+#     - make it ignore temporary files generated by dev processes, e.g. coverage checks
+#     - encode typical decisions, e.g. whether we package docs, requirements etc
+
+#---------
+# SECTION: main code sources
+#
 recursive-include src *.py *.pyx
+    # principles:
+    #    - *no* "prune" command is used
+    #    - use "include-recursive", by relevant file extensions
+    # hints:
+    #     - top-level dir is typically "src", but maybe "lib" or <package-name>
+    #     - default package rules mean we generally don't actually *need* a statement
+    #         - but it's better to be explicit
+    #     - extension filetypes are typically sources (*.py)
+    #         - might also need testdata files, e.g. *.nc, *.npy *.npz
+    #         - also possibly non-python, e.g. *.pyx for Cython
+
 
-include .git_archival.txt
-include .gitattributes
+#---------
+# SECTION: requirements
+prune requirements
+recursive-include requirements *.txt
+    # principles:
+    #    include just requirements-level info, not lock files
+    # hints:
+    #    - not all projects include requirements, but they can be drawn in anyway by dynamic dependencies
+    #        in the setuptools build process, linked via config in pyproject.toml
+    #    - for some repos, the *.txt are actually lockfiles.  This probably needs fixing.
```

Comment:
As well as dropping the docs section, SciTools/.github#181 has also removed this line.